### DISCUSSION TITLE
Upgrade the markdown version so we can add target="_blank"

### DISCUSSION THREAD

**Discussion by KarelJanVanHaute** [craftcms/cms#13327] [Ideas] [answered]
In newer versions of markdown, this should be possible
```
[link](URL){:target="_blank"}
```
For as now, it doesn't seem possible in Craft. 
It would be handy to link to another global, and to have that open up in a new tab.

**Comment by brandonkelly** [comment, marked as answer]
That’s not currently part of the CommonMark spec (latest at time of writing is [0.30](https://spec.commonmark.org/0.30/)).

It has been discussed [here](https://talk.commonmark.org/t/consistent-attribute-syntax/272/143) though the conversation has mostly fizzled away, it seems.

If it does end up making it in, I’m sure it will be supported by [league/commonmark](https://commonmark.thephpleague.com/), which we plan to start using for Markdown parsing in Craft 5.
